sdk/storage/storage-blob/test/blobbatch.spec.ts

```diff
@@ -413,6 +413,97 @@ describe("BlobBatch", () => {
     }
   });
 
+  it("submitBatch should work for batch set tier with versioning", async () => {
+    recorder.skip(
+      undefined,
+      "UUID is randomly generated within the SDK and used in the HTTP request and cannot be preserved."
```

Comment:
Not blocking PR but I wonder whether @HarshaNalluru has any ideas on potential improvement from test-utils-recorder side.

Reply:
Hmmm, no bright ideas. We have no way to override since it is generated within the SDK.
Long back while discussing, came to know that the Java SDK also has the same problem where they skip such tests where `uuid`s are generated within the SDK.

That being said, I don't think we have an issue to track this, logging now for further investigation.
(Possibly mapping all `uuid`s to fixed values in the requests and responses in sequence or overriding them before they are used once generated)